src/main/resources/user.avsc

```diff
@@ -0,0 +1,9 @@
+{"namespace": "example.avro",
```

Comment:
Detta var en middlertidig fil som jag senare tog bort. Trenger inte att ta den in igen :)